documentation/cybersecurity.workspace.adoc

```diff
@@ -0,0 +1,481 @@
+
+The field is becoming increasingly significant due to the  ever-expanding reliance on computer systems, the Internet and wireless network standards such as Bluetooth and Wi-Fi, and due to the growth of "smart" devices. 
+
+Cybersecurity is also one of the significant challenges in the contemporary world, due to its complexity, both in terms of political usage and technology.
+
+////
+Losses attributed to cybercrime are expected to reach $10.5 trillion by 2025*, growing at a rate of 15 percent annually. Costs include damage and destruction of data, stolen money, theft of intellectual property, personal and financial data, investigation, restoration, and more. While the digital economy is flourishing, 68 percent of business leaders recognize their cybersecurity risks are also increasing, urging many to invest in the right tools. Gartner predicts the global information security market will reach a staggering $170.4 billion USD by 2022*.
+////
+
+Graph databases can help in implementing an effective solution in the field of Cyber Security.
+
+[quote, John Lambert, Engineer from Microsoft Threat Intelligence Center]
+Defenders think in lists. Attackers think in graphs. As long as this is true, attackers win.
+
+
+Our networks are *connected graphs*.
+We can use a graph database-based solution to get a holistic view of enterprise network. 
+
+image::{img}/example.svg[width=470]
+
+This helps us to
+
+1. quickly identify risks,
+2. detecting anomalies and 
+3. protecting our systems with confidence 
+
+to minimize impacts.
+
+== Demo Use-case
+
+image::{img}/bloodhound.png[float=left,link="https://bloodhound.readthedocs.io/en/latest/",window="_blank"]
+
+This demo is based on the data and themes from the BloodHound project. 
+Bloodhound is a great tool for auditing your Active Directory environment. 
+It uses Neo4j as a backing datastore for storage and querying. 
+We recommend looking into BloodHound or reading the very comprehensive handbook on it to learn more about BloodHound.
+
+
+* https://bloodhound.readthedocs.io/en/latest/[BloodHound Documentation^]
+* https://ernw.de/download/BloodHoundWorkshop/ERNW_DogWhispererHandbook.pdf[BloodHound Handbook^]
+
+We have conceptualized a BloodHound example to prepare a fictitious Active Directory environment and showed some useful analysis.
+
+Active directory helps IT teams monitor various network resources and users. 
+It allows to grant and revoke different user permissions.
+
+Active Directory comprises of various objects. 
+We have considered below sample objects for this use case:
+
+* User
+* Computer
+* Group
+* Domain
+* Group Policy Object (GPO) - Virtual Collection of Policy Settings
+* OU (Organization Unit) - Sub division
+
+Details:
+
+Nodes {nodes} Relationships {relationships}
+
+.Model
+image::{img}/model.svg[width=470]
+
+// ifndef::env-guide[]
```

Comment:
you can remove this whole section,
btw with `////`  you can do block comments as show above.